docs/index.md

```diff
@@ -36,7 +36,7 @@ load("@io_bazel_rules_rust//:workspace.bzl", "rust_workspace")
 rust_workspace()
 ```
 
-The rules are under active development, as such the lastest commit on the master branch should be used. `master` currently requires Bazel >= 0.26.0.
+The rules are under active development, as such the lastest commit on the master branch should be used. `master` currently requires Bazel >= 3.0.0.
```

Comment:
I think https://github.com/bazelbuild/rules_rust/blob/master/workspace.bzl#L5 should be updated too.